packages/cms/pages/auth/register/register.tsx

```diff
@@ -12,12 +12,30 @@ import { EuiFlexGroup } from '@tensei/eui/lib/components/flex'
 import { EuiFieldText } from '@tensei/eui/lib/components/form/field_text'
 import { EuiFieldPassword } from '@tensei/eui/lib/components/form/field_password'
 import { EuiFlexItem } from '@tensei/eui/lib/components/flex/flex_item'
+import { useState } from 'react'
 
 const H3 = styled.h3`
   text-align: center;
 `
 
 export const Register: React.FunctionComponent = () => {
+  const [user, setUser] = useState({
+    firstname: '',
+    lastname: '',
+    email: '',
+    password: ''
+  })
+
+  const submitRegistration = async (e: any) => {
```

Comment:
A better variable name would be `event` or `submitEvent` and the type would be `(event: React.FormEvent<HtmlFormElement>)`

Reply:
Thanks for this